lib/app.rb

```diff
@@ -153,4 +150,20 @@ def run
       command = @input_stream.gets.chomp
     end
   end
+
+  private
+
+  def switch_current_project
```

Comment:
My general idea is to keep these as method, extract other things to methods as I move forward, then see what objects fall out. Is that not bold enough and I should go ahead now with making objects?

Reply:
That seems like the right refactoring plan for this stage of the project. There is a bunch of work to do in understanding the code before you can build up the right abstractions. Waiting to see what pops out is exactly right!